machine/v1/tests/controlplanemachinesets.machine.openshift.io/CMPSMachineNamePrefix.yaml

```diff
@@ -0,0 +1,134 @@
+apiVersion: apiextensions.k8s.io/v1 # Hack because controller-gen complains if we don't have this
+name: "ControlPlaneMachineSet (+CMPSMachineNamePrefix)"
```

Comment:
All occurrences of `CMPS` should be `CPMS`

Reply:
Not sure why I have written everywhere `CMPS` instead of `CPMS`, my bad :(  Corrected all of them.